test/src/main/java/org/apache/accumulo/test/compaction/ExternalCompactionMetricsIT.java

```diff
@@ -163,7 +176,9 @@ public void testMetrics() throws Exception {
       // Wait for all external compactions to complete
       long count;
       do {
-        UtilWaitThread.sleep(100);
+        // TODO: Change this from waiting to verifying that all compactors are done running jobs,
```

Comment:
This seems like a good change, were you thinking of making the change in this PR?

Reply:
No, I want to add that change in a different PR.